frontend/packages/console-app/src/components/quick-starts/data/serverless-application-quickstart.ts

```diff
@@ -26,6 +26,7 @@ export const serverlessApplicationQuickStart: QuickStart = {
 4. In the **Git Repo URL** field, type \`https://github.com/sclorg/django-ex.git\`.
 5. Under Resources, select **Knative Service**.
 6. At the end of the page, click **Create**.
+8. test extension: [link]{{highlight tour-search-nav}}
```

Comment:
Not sure what our mechanism will be for the link... we may need to work with UX next week on that... But I'm currently thinking this example was akin to `[data-tour-id="tour-search-nav"]`... that probably isn't extensible enough, but like I said, we can circle back to that.